synapse/push/baserules.py

```diff
@@ -19,7 +19,7 @@
 from synapse.push.rulekinds import PRIORITY_CLASS_INVERSE_MAP, PRIORITY_CLASS_MAP
 
 
-def list_with_base_rules(rawrules):
+def list_with_base_rules(rawrules, use_new_defaults=False):
```

Comment:
I know it's supposed to be temporary, but that's what we said about the whole of synapse 5 years ago. Can you give `use_new_defaults` a docstring?